tests/pamiq_core/utils/test_reflection.py

```diff
@@ -0,0 +1,14 @@
+from pamiq_core.utils import get_class_module_path
+
+
+def test_get_class_module_path():
+    class TestClass:
+        pass
```

Comment:
これはモジュールスコープにおくべきですね。
```suggestion
class TestClass:
    pass

def test_get_class_module_path():
```

Reply:
この方向で修正しました！